modules/drivers/lidar_surestar/include/component/component_driver.h

```diff
@@ -0,0 +1,103 @@
+/******************************************************************************
```

Comment:
Better restructure the folders.  Directories like `include`, `lib` are meaningless under Bazel.Build.

Reply:
Can you please make it clear what structure should be used for Bazel build system?

Reply:
> clear what structure should be used for Bazel build system?

As you can see from other bazel examples,  `a.cc` and `a.h` are usually placed in the same directory. So we'd better not using 'include'/'lib' as a path component. Say, we have 4 c++ files, we should directly write 'garden/flower.{h,cc},grass.{h,cc}`, rather than `garden/include/{flower,grass}.h` and `garden/lib/{flower,grass}.cc`.

Hopes that I have made my point clear.